### PR TITLE
Add documentation on how to update a tagged container

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: docs/how-to-build.md [hunk 1]
```diff
@@ -13,3 +13,17 @@ git push --tags
...
 
 The build system watches for tags which begin with a 7 for puppet7-release and 8 for puppet8-release.
+
+# Update a container
+
+If you want to rebuild a container, or apply some updates from the build system, you can do it like this:
+
+```bash
+git pull -r
+git tag -d 7.13.0
```
Comment (bastelfreak) [inline docs/how-to-build.md:23]: I don't think we should modify tags. This will create conflicts for people that have the old tag locally.

**Review by bastelfreak — CHANGES_REQUESTED** [review]

File: docs/how-to-build.md (hunk 1 above)
Comment (rwaffen) [reply docs/how-to-build.md:23]: we shouldn't do this, yes, but if we have to, i wanted to have documented how you have to do it.

Comment (rwaffen) [reply docs/how-to-build.md:23]: maybe we should use a version file, and get the version from there 🤔 ... we rely somewhat to much on the tag name here

Comment (tuxmea) [reply docs/how-to-build.md:23]: I prefer to use the tags to identify Puppet version to put inside containers.
I don't like the idea to re-use existing tags and point them to new commits.

How about the following idea:

- tag 7.13.1 builds with the latest available version 7.13.1 from puppet.
- If puppet release is at 7.13.1-1 this version is used at initial build time.

If you want to update the container, after puppet has released 7.13.1-2 one can start from tag 7.13.1 (`git checkout 7.13.1`) do the required adoptions and push a new tag (`git tag 7.13.1-2`)

Comment (bastelfreak) [reply docs/how-to-build.md:23]: I agree with @tuxmea here. I think we should never manipulate the history and instead use a pkgrel style and append -1, then -2 etc.

Comment (rwaffen) [reply docs/how-to-build.md:23]: lets discuss this in #5

**Pull request closed without merging**